appveyor.yml

```diff
@@ -1,33 +1,37 @@
 # version is set in tools\appveyor.psm1 - Invoke-AppVeyorInstall
+environment:
+  POWERSHELL_TELEMETRY_OPTOUT: 1
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+  matrix:
+  - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+    Purpose: BuildingAndPackaging
+  - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+    Purpose: BuildingAndTesting
 
-image: Visual Studio 2017
+# Stop all jobs in the matrix if any one them fail
+matrix:
+  fast_finish: true
 
-# cache version - netcoreapp.2.1-sdk.2.1.300-rc1
 cache:
-  - '%LocalAppData%\Microsoft\dotnet -> appveyor.yml'
```

Comment:
This is here for reliability.  Please don't remove

Reply:
Are you saying the download of the SDK sometimes fails? Usually cache makes things less reliable due to leftovers. The reason for removal was the amount of time it was taking and the fact that it often also fails due to its size.

Reply:
Ok, but then shouldn't we rather cache just what we have downloaded instead of caching the appdata folder of dotnet?